### PR TITLE
[NUI] DragAndDrop : change drag window reusable

### PR DESCRIPTION
**Pull request by taehyub**
### Description of Change ###

change drag window resuable to improve performance

### API Changes ###
N/A

### CODE REVIEW

**Review by everLEEst — COMMENTED** [review]
it seems ACR patch and I don't know why bot attached not acr required tag... if it is right to be ACR, please add ACR Required tag manually.

File: src/Tizen.NUI/src/public/DragAndDrop/DragAndDrop.cs [hunk 1]
```diff
@@ -62,46 +65,37 @@ private DragAndDrop(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cM
...
+        /// <since_tizen> 10 </since_tizen>
         public void StartDragAndDrop(View sourceView, View shadowView, DragData dragData, SourceEventHandler callback)
         {
             if (null == shadowView)
             {
                 throw new ArgumentNullException(nameof(shadowView));
             }
 
-            if (mDragWindow)
+            if (!mDragWindow)
```
Comment (jaehyun0cho) [inline src/Tizen.NUI/src/public/DragAndDrop/DragAndDrop.cs:76]: I think null check is correct for this.
if (mDragWindow != null)

File: src/Tizen.NUI/src/public/DragAndDrop/DragAndDrop.cs [hunk 2]
```diff
@@ -62,46 +65,37 @@ private DragAndDrop(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cM
...
             {
-                mDragWindow.Hide();
-                mDragWindow.Dispose();
-                mDragWindow = null;
+              mDragWindow = new Window("DragWindow", new Rectangle(-shadowWidth, -shadowHeight, shadowWidth, shadowHeight), true)
+              {
+                BackgroundColor = Color.Transparent,
```
Comment (jaehyun0cho) [inline src/Tizen.NUI/src/public/DragAndDrop/DragAndDrop.cs:80]: Please check indentation

**Review by jaehyun0cho — APPROVED** [review]
LGTM

**Comment by dongsug-song** [comment]
@taehyub 
안녕하세요.
acr 이 accept 되어야만 머지가능 합니다.
그리고, 
/// <since_tizen>10</since_tizen  => 추가
[EditorBrowsable(EditorBrowsableState.Never)]  ==> 제거 
로 고쳐주셔야 "acr required" tag 가 붙습니다. 
그리고나서, acr accept 가 되면 review 및 merge 가 됩니다.
감사합니다.

**Pull request merged**